packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -25,6 +25,7 @@ export const getCheckboxStyles = stylesFactory((theme: GrafanaTheme) => {
     description: cx(
       labelStyles.description,
       css`
+        line-height: 1.25px;
```

Comment:
```suggestion
        line-height: 1.25;
```

Reply:
was thinking if we should use the line heights in the `theme.typography.lineHeight` ?

Reply:
We don't have `line-height` 1.25 in the themes, the closest is 1.1, which also works I think.

Reply:
I'd suggest using that so we don't introduce custom line-heights

Reply:
Thanks for the pointer. I'll fix that